pandas/tests/groupby/test_counting.py

```diff
@@ -195,3 +196,28 @@ def test_ngroup_respects_groupby_order(self):
                                 g.ngroup())
             assert_series_equal(Series(df['group_index'].values),
                                 g.cumcount())
+
+    def test_count_with_datetimelike(self):
+        # test for #13393, where DataframeGroupBy.count() fails
+        # when counting a datetimelike column.
+
+        expected = DataFrame({'y': [2, 1]}, index=['a', 'b'])
+        expected.index.name = "x"
+
```

Comment:
use parametrize (the decorator)

Reply:
Oh! I looked in the wrong test file for inspiration then.